io.catenax.request_for_quotation/1.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,669 @@
+
+:postProcessGroup a samm:Property ;
+   samm:preferredName "post process group"@en ;
+   samm:description "process group is depending on the defined manufacturing method and defines the general treatment"@en ;
+   samm:characteristic :PostProcessGroupCharacteristic ;
+   samm:exampleValue "heat treatment, surface treatment, surface finishing..." .
+
+:postProcessType a samm:Property ;
+   samm:preferredName "post process type"@en ;
+   samm:description "specific process out of the process group"@en ;
+   samm:characteristic :PostProcessTypeCharacteristic ;
+   samm:exampleValue "stress releaving, sandblasting, colouring" .
+
+:postProcessProperty a samm:Property ;
+   samm:preferredName "post process property"@en ;
+   samm:description "additional property based on the post process type "@en ;
+   samm:characteristic :PostProcessPropertyCharacteristic ;
+   samm:exampleValue "black (for colouring)" .
+
+:quantityNumber a samm:Property ;
+   samm:preferredName "Quantity Number"@en ;
+   samm:description "The number of objects related to the measurement unit"@en ;
+   samm:characteristic :NumberOfObjects ;
+   samm:exampleValue "2.5"^^xsd:double .
+
+:measurementUnit a samm:Property ;
+   samm:description "Unit of Measurement for the quantity of serialized objects"@en ;
+   samm:see <https://eclipse-esmf.github.io/samm-specification/2.0.0/appendix/unitcatalog.html> ;
+   samm:characteristic samm-c:UnitReference ;
+   samm:exampleValue "unit:litre"^^samm:curie .
+
+:materialItems a samm:Property ;
+   samm:preferredName "material items"@en ;
+   samm:description "material definition with additional detail elements (name, density, chemical name,...)"@en ;
+   samm:characteristic :SetMaterialItemsCharacteristic .
+
+:materialGroup a samm:Property ;
+   samm:preferredName "material group"@en ;
+   samm:description "definition of material group for preselection"@en ;
+   samm:characteristic :PartMaterialGroupCharacteristic ;
+   samm:exampleValue "aluminum, steel, ..." .
+
+:createdOn a samm:Property ;
+   samm:preferredName "Created On"@en ;
+   samm:description "Timestamp when the relation between the parent item and the child item was created, e.g. when the serialized child part was assembled into the given part."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime .
+
+:lastModifiedOn a samm:Property ;
+   samm:preferredName "Last Modification Date"@en ;
+   samm:description "Timestamp when the assembly relationship between parent item and child item was last modified."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime .
+
+:businessPartner a samm:Property ;
+   samm:preferredName "Business Partner"@en ;
+   samm:description "The supplier of the given child item."@en ;
+   samm:characteristic :BpnTrait ;
```

Comment:
You can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR104

Reply:
Thank you for the hint!
I removed the local definition and referenced the `urn:samm:io.catenax.shared.business_partner_number` model